service/src/main/java/bio/terra/drshub/services/RestTemplateFactory.java

```diff
@@ -38,7 +38,7 @@ public RestTemplate makeMTlsRestTemplateWithPooling(MTlsConfig mTlsConfig) {
     var keyManager =
         PemUtils.loadIdentityMaterial(mTlsConfig.getCertPath(), mTlsConfig.getKeyPath());
     var sslFactory = SSLFactory.builder().withIdentityMaterial(keyManager).build();
-    var socketFactory = Apache5SslUtils.toSocketFactory(sslFactory);
+    var socketFactory = Apache5SslUtils.toTlsSocketStrategy(sslFactory);
```

Comment:
This method (`toSocketFactory`) was removed in the update from `sslcontext-kickstart-for-apache5` -> `ayza-for-apache5` and replaced with `toTlsSocketStrategy` to align with the deprecation of `PoolingHttpClientConnectionManagerBuilder.setSSLSocketFactory` call on line 56 which is replaced with  `PoolingHttpClientConnectionManagerBuilder.setTlsSocketStrategy`

Reply:
Apache5 docs for the deprecation of `setSSLSocketFactory`:
https://hc.apache.org/httpcomponents-client-5.5.x/current/httpclient5/apidocs/org/apache/hc/client5/http/impl/io/PoolingHttpClientConnectionManagerBuilder.html#setSSLSocketFactory-org.apache.hc.client5.http.socket.LayeredConnectionSocketFactory-